### PR TITLE
GCI : Task > PostGIS: Fix some strings in pgAdmin4 geometry viewer > …

### PR DESCRIPTION
**Pull request by thisIsShubhamSharma**
PostGIS: Fix some strings in pgAdmin4 geometry viewer…ShubhamSharama

### CODE REVIEW

File: web/pgadmin/static/js/sqleditor/geometry_viewer.js [hunk 1]
```diff
@@ -326,7 +326,7 @@ function parseData(items, columns, columnIndex) {
 
   // generate map info content
   if (tooLargeDataSize || tooManyGeometries) {
-    infoList.push(supportedGeometries.length + ' of ' + items.length + ' geometries rendered.');
+    infoList.push(supportedGeometries.length + gettext(' of ') + items.length + gettext(' geometries rendered.'));
```
Comment (Gooong) [inline web/pgadmin/static/js/sqleditor/geometry_viewer.js]: This won't work with some languages. You can build the string as: `gettext("%d of %d geometries rendered." %(supportedGeometries.length, items.length))`

File: web/pgadmin/static/js/sqleditor/geometry_viewer.js [hunk 2]
```diff
@@ -372,7 +372,7 @@ function parseData(items, columns, columnIndex) {
   }
 
   if (mixedSRID) {
-    infoList.push(gettext('Geometries with non-SRID') + selectedSRID + ' not rendered.');
+    infoList.push(gettext('Geometries with non-SRID') + selectedSRID + gettext(' not rendered.'));
```
Comment (Gooong) [inline web/pgadmin/static/js/sqleditor/geometry_viewer.js]: Same as above

**Comment by thisIsShubhamSharma** [comment]
Thankx a lot really appreciate it.

**Comment by thisIsShubhamSharma** [comment]
Will surely do.

**Comment by thisIsShubhamSharma** [comment]
Please review again and thank you for your help.

**Pull request closed without merging**